web/src/components/molecules/Asset/Asset/AssetBody/previewToolbar.tsx

```diff
@@ -1,12 +1,13 @@
 import Button from "@reearth-cms/components/atoms/Button";
 import Icon from "@reearth-cms/components/atoms/Icon";
 import PreviewModal from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/previewModal";
-import { PreviewType } from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/previewTypeSelect";
+import { PreviewType } from "@reearth-cms/gql/graphql-client-api";
 
 type Props = {
   url: string;
-  selectedPreviewType: string;
+  selectedPreviewType: PreviewType;
   isModalVisible: boolean;
+  isSVG: boolean;
```

Comment:
Unless this can't be undefined due to a mutation's type or something, if you make this optional it makes using the component a little bit nicer (then you can avoid having to pass isSVG even when it isn't).